File: tuts/003-s3-gettingstarted/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_s3 "github.com/aws/aws-sdk-go-v2/service/s3"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_s3.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running S3 getting started scenario...")
	// TODO: setup, interact, teardown
}

File: tuts/003-s3-gettingstarted/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { S3Client } from "@aws-sdk/client-s3";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new S3Client({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running s3 getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();
